### DISCUSSION TITLE
Pagina web

### DISCUSSION THREAD

**Discussion by WinstonTorres24** [community/community#65789] [Programming Help]
Buenas tardes, estaba desarrollando una sitio Web, en Visual studio code "Utilizando Bootstrap", pero cuando voy a ver el Deployments, me muestra la pagina subida pero sin las características que aplique en la pagina
Este es el Deployed de la pagina:
https://winstontorres24.github.io/
Este es el Repositorio:
[WinstonTorres24.github.io](https://github.com/WinstonTorres24/WinstonTorres24.github.io)]
gracias.

**Comment by seyLu** [comment]
#### Hi  @WinstonTorres24,

#### Domain name is case-insensitive but the rest of the url is not.

![image](https://github.com/community/community/assets/98249191/f30d282a-244e-4783-a5c0-c92d8632ac6f)
![image](https://github.com/community/community/assets/98249191/3695a1ba-d2d9-4b23-8d08-52a48c8a8a41)

#### You can read more on: [Should url be case-sensitive](https://stackoverflow.com/questions/7996919/should-url-be-case-sensitive).

**Comment by WinstonTorres24** [comment]
Hola MJ Sabit, pero no entiendo cuál es el provlema o como solucionarlo

**Reply by albedim** [reply]
Try to write css instead of CSS in the url.
